components/Pagination/PaginationItem/PaginationItem.js

```diff
@@ -30,7 +36,7 @@ function PaginationItem({ children, isCurrent, pathname, testId, value }) {
       data-testid={testId}
     >
       {isClickable ? (
-        <Link href={pathname} as={pathname.replace('[page]', value)}>
+        <Link href={{ pathname, query }} as={realURL}>
```

Comment:
just for full clarification: **href** = path in the directory, **as** = just the "aesthetic" of what shows in the browser? and if so then why can you not make it whatever you want? I was getting errors about the `href` and `as` path not being compatible

Reply:
`as` is only a prop for dynamic routes. Typically you can just pass `href`, but since `Next.js` does route matching via the file system... dynamic routes `[page]` wouldn't map nicely to a URL.

`resources/index.js` -> /resources
`resources/1.js` -> /resources/1
`resources/[page].js` -> is dynamic and needs to be handled differently than just "trust the file system"